spec/controllers/conferences_controller_spec.rb

```diff
@@ -54,4 +54,16 @@
       end
     end
   end
+
+  describe 'POST create' do
+    context 'unauthenticated users' do
+      let(:params) { { name: 'name', country: 'country', region: 'region', location: 'location', city: 'city', season_id:'1', url: 'www.conference.com' } }
```

Comment:
What's the point in assigning this huge params hash if you in the end only want to test whether or not a user is signed in? I'd suggest to shorten it to:
```ruby
it "does not create a new conference record" do
  expect { post :create, params: conference: {} }.not_to change { Conference.count }
end
```

Reply:
I thought this would isolate the behavior I'm looking at, so that the test won't fail because the parameters are invalid, but because the user is not logged in.

Reply:
Valid point!

I don't know the `conference` factory from the top of my head, but eventually `attributes_for :conference` would be shorter and saver 🤔 

Maybe the test would be also be more meaningful if you'd also look at the response status? As in: if it's a `401`, we know the request was no authorized.